<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/FAX.png</normaloff>:/icons/icons/FAX.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Deliver"/>
   <addaction name="action_Return"/>
   <addaction name="action_Receipt"/>
   <addaction name="separator"/>
   <addaction name="action_DeliveryRecords"/>
   <addaction name="action_ReceiptRecords"/>
   <addaction name="separator"/>
   <addaction name="action_DeliverySummarize"/>
   <addaction name="action_ReceiptSummarize"/>
   <addaction name="action_DeliveryReport"/>
   <addaction name="action_ReceiptReport"/>
   <addaction name="separator"/>
   <addaction name="action_Customers"/>
   <addaction name="action_Products"/>
   <addaction name="action_Employees"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_System">
    <property name="title">
     <string>系统 (&amp;S)</string>
    </property>
    <addaction name="action_Backup"/>
    <addaction name="action_Restore"/>
    <addaction name="separator"/>
    <addaction name="action_DataImport"/>
    <addaction name="action_DataExport"/>
    <addaction name="separator"/>
    <addaction name="action_Changepassword"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Delivery">
    <property name="title">
     <string>发货 (&amp;D)</string>
    </property>
    <addaction name="action_Deliver"/>
    <addaction name="action_Return"/>
    <addaction name="separator"/>
    <addaction name="action_DeliveryRecords"/>
    <addaction name="action_DeliverySummarize"/>
   </widget>
   <widget class="QMenu" name="menu_Receipt">
    <property name="title">
     <string>收款 (&amp;F)</string>
    </property>
    <addaction name="action_Receipt"/>
    <addaction name="separator"/>
    <addaction name="action_ReceiptSummarize"/>
    <addaction name="action_ReceiptSheet"/>
   </widget>
   <widget class="QMenu" name="menu_Meta">
    <property name="title">
     <string>基础资料 (&amp;M)</string>
    </property>
    <addaction name="action_Customers"/>
    <addaction name="action_Products"/>
    <addaction name="action_Employees"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助 (&amp;H)</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Report">
    <property name="title">
     <string>报表(&amp;R)</string>
    </property>
    <addaction name="action_DeliveryReport"/>
    <addaction name="action_ReceiptReport"/>
   </widget>
   <addaction name="menu_System"/>
   <addaction name="menu_Delivery"/>
   <addaction name="menu_Receipt"/>
   <addaction name="menu_Meta"/>
   <addaction name="menu_Report"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Backup">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_135_inbox_out.png</normaloff>:/icons/icons/glyphicons_135_inbox_out.png</iconset>
   </property>
   <property name="text">
    <string>备份数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_Restore">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_134_inbox_in.png</normaloff>:/icons/icons/glyphicons_134_inbox_in.png</iconset>
   </property>
   <property name="text">
    <string>恢复数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="action_Changepassword">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_240_rotation_lock.png</normaloff>:/icons/icons/glyphicons_240_rotation_lock.png</iconset>
   </property>
   <property name="text">
    <string>更改密码</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_063_power.png</normaloff>:/icons/icons/glyphicons_063_power.png</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
  <action name="action_Deliver">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_208_cart_out.png</normaloff>:/icons/icons/glyphicons_208_cart_out.png</iconset>
   </property>
   <property name="text">
    <string>发货</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Return">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_209_cart_in.png</normaloff>:/icons/icons/glyphicons_209_cart_in.png</iconset>
   </property>
   <property name="text">
    <string>退货</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_DeliveryRecords">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_114_list.png</normaloff>:/icons/icons/glyphicons_114_list.png</iconset>
   </property>
   <property name="text">
    <string>发货记录</string>
   </property>
  </action>
  <action name="action_DeliverySummarize">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_089_building.png</normaloff>:/icons/icons/glyphicons_089_building.png</iconset>
   </property>
   <property name="text">
    <string>发货汇总</string>
   </property>
  </action>
  <action name="action_Receipt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_227_usd.png</normaloff>:/icons/icons/glyphicons_227_usd.png</iconset>
   </property>
   <property name="text">
    <string>收款</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_ReceiptSummarize">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_325_wallet.png</normaloff>:/icons/icons/glyphicons_325_wallet.png</iconset>
   </property>
   <property name="text">
    <string>收款汇总</string>
   </property>
  </action>
  <action name="action_ReceiptSheet">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_087_log_book.png</normaloff>:/icons/icons/glyphicons_087_log_book.png</iconset>
   </property>
   <property name="text">
    <string>对账</string>
   </property>
  </action>
  <action name="action_Customers">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_024_parents.png</normaloff>:/icons/icons/glyphicons_024_parents.png</iconset>
   </property>
   <property name="text">
    <string>客户资料</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Products">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_149_folder_new.png</normaloff>:/icons/icons/glyphicons_149_folder_new.png</iconset>
   </property>
   <property name="text">
    <string>产品资料</string>
   </property>
   <property name="toolTip">
    <string>产品资料</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Employees">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_043_group.png</normaloff>:/icons/icons/glyphicons_043_group.png</iconset>
   </property>
   <property name="text">
    <string>业务资料</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_DeliveryReport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_036_file.png</normaloff>:/icons/icons/glyphicons_036_file.png</iconset>
   </property>
   <property name="text">
    <string>发货报表</string>
   </property>
  </action>
  <action name="action_ReceiptReport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_323_calculator.png</normaloff>:/icons/icons/glyphicons_323_calculator.png</iconset>
   </property>
   <property name="text">
    <string>应收报表</string>
   </property>
  </action>
  <action name="action_DataImport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_221_unshare.png</normaloff>:/icons/icons/glyphicons_221_unshare.png</iconset>
   </property>
   <property name="text">
    <string>数据导入</string>
   </property>
  </action>
  <action name="action_DataExport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_222_share.png</normaloff>:/icons/icons/glyphicons_222_share.png</iconset>
   </property>
   <property name="text">
    <string>数据导出</string>
   </property>
  </action>
  <action name="action_ReceiptRecords">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_113_justify.png</normaloff>:/icons/icons/glyphicons_113_justify.png</iconset>
   </property>
   <property name="text">
    <string>收款记录</string>
   </property>
   <property name="toolTip">
    <string>收款记录</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/icons/glyphicons_195_circle_info.png</normaloff>:/icons/icons/glyphicons_195_circle_info.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
